index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+
+    <script src="lib/jquery-2.1.4.min.js"></script>
+    <script src="main.js"></script>
+
```

Comment:
We never use inline css or <style> always include a .css file. Please move it out